src/lib/addVideoObjectMarkup.js

```diff
@@ -0,0 +1,129 @@
+/**
+ * Append special 'vimeo_t' time param to a URL
+ *
+ * @param {number} time Start time of video chapter
+ * @return {string}
+ */
+function appendTimeQuery(time) {
+    const { href, search, hash } = window.location;
+    let newHref = '';
+    const timeParam = `vimeo_t=${time}`;
+
+    if (search.length) {
+        const qs = `${search}&${timeParam}`;
+        newHref = href.replace(search, qs);
+    }
+    else {
+        newHref = hash.length ? href.replace(hash, `?${timeParam}${hash}`) : `${href}?${timeParam}${hash}`;
+    }
+
+    return newHref;
+}
```

Comment:
ESLint informed me that the URL API is not IE11 compatible, so I created this appendTimeQuery helper method. I tested it against URLs like:
```
https://www.example.com
https://www.example.com#t
https://www.example.com?q1=33&q2=99
https://www.example.com?q1=33&q2=99#t
```
I'm trying to think of edge cases for which this method might not work.